database/migrations/2019_08_01_000001_create_domain_verifications_table.php

```diff
@@ -15,11 +15,16 @@ public function up()
     {
         Schema::create('domain_verifications', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('verifiable_id')->index();
+            $table->unsignedBigInteger('verifiable_id');
+            $table->string('verifiable_type');
             $table->string('url')->index();
             $table->string('token')->index();
+            $table->string('status');
```

Comment:
Chuyển cái này qua thành enum thì hơn e nhỉ?

```suggestion
            $table->enum('status');
```

Reply:
Ok ạ